gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -1830,6 +1842,17 @@ _gtfc_setprop_OUTPUTCOMBINATION (GstTensorFilterPrivate * priv,
   return ret;
 }
 
+/** @brief Handle "PROP_SHARED_TENSOR_FILTER_KEY" for set-property */
+static gint
+_gtfc_setprop_SHARED_TENSOR_FILTER_KEY (GstTensorFilterProperties * prop,
+    const GValue * value)
+{
+  const gchar *sKey = g_value_get_string (value);
+  prop->shared_tensor_filter_key = g_strdup (sKey);
```

Comment:
`prop->shared_tensor_filter_key = g_value_dup_string (value)` is simple :)

Reply:
Memory leak from ```prop->shared_tensor_filter_key```.
Do ```g_free()``` before calling ```g_strdup()```

Reply:
I applied both.